grade.go

```diff
@@ -301,11 +280,10 @@ func OSReadDir(root string) []string {
 }
 
 // Parse user input flags and return as strings.
-func parseFlags() (workDir, runArgs, outFile, inFile, language string, wall bool, isDryRun bool, server bool, assignmentId int) {
+func parseFlags() (workDir, runArgs, outFile, language string, wall bool, isDryRun bool, server bool, assignmentId int) {
 	flag.StringVar(&workDir, "directory", "/code", "student submissions directory")
 	flag.StringVar(&runArgs, "args", "", "arguments to pass to compiled programs")
 	flag.BoolVar(&wall, "Wall", true, "compile programs using -Wall")
-	flag.StringVar(&inFile, "in", "", "file to read interactive input from")
```

Comment:
should probably also remove outFile since that's also defined in `Test`

Reply:
(I think)

Reply:
outFile is the `.csv` file to write the results to, so its not defined in test--but it easily can be if we want to

Reply:
ahh makes sense- didn't read what it was for